core-bundle/tests/Routing/RouteProviderTest.php

```diff
@@ -662,9 +671,11 @@ private function mockFramework(Adapter $pageAdapter = null, Adapter $configAdapt
      */
     private function createPage(string $language, string $alias, bool $fallback = true, string $domain = '', string $scheme = null): PageModel
     {
+        mt_srand(++$this->pageModelAutoIncrement);
```

Comment:
why do we seed the random number generator here? We seed it again in randomBytes?

Reply:
For the line `$page->rootSorting = mt_rand();` further down. I think @leofeyer moved the line up here in [`a54cea5` (#2879)](https://github.com/contao/contao/pull/2879/commits/a54cea5e9396faaf85fa615c7f7dc23bfcae8bab)